integration-libs/opf/checkout/root/config/default-opf-routing-config.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { RoutingConfig } from '@spartacus/core';
+
+export const defaultOpfRoutingConfig: RoutingConfig = {
+  routing: {
+    routes: {
+      paymentVerificationResult: {
+        paths: ['redirect/success'],
+      },
+      paymentVerificationCancel: {
+        paths: ['redirect/failure'],
+      },
```

Comment:
[MEDIUM]

Is the difference between the semantinc url name and the actual URL necessary?
- key `...Result` vs value `...successs`
- key `...Cancel` vs value `...failure`

For future easier reasoning and discoverability of the "URLs vs semantic names", can we align a bit the keys with the values?